File: backEnd/platform-automation/src/main/java/com/mokatest/platform/demos/domain/ui/dto/step/SwitchTabStepDTO.java
package com.mokatest.platform.demos.domain.ui.dto.step;

// 切换页签

import com.mokatest.platform.demos.domain.ui.uiEnum.browser.SwitchTabMode;
import lombok.Data;

@Data
public class SwitchTabStepDTO extends StepBaseDTO {
    private String stepType;
    private SwitchTabMode switchTabMode; // 关闭页面模式
    private Integer customIndex; // 自定义关闭页面索引
}

File: frontEnd/src/types/dto/stepDTO/SwitchTabStepDTO.ts
import {SwitchTabMode} from "@/types/enum/browser/SwitchTabMode";
import {StepBaseDTO} from "@/types/dto/base/StepBaseDTO";

// 切换页签
export class SwitchTabStepDTO extends StepBaseDTO {
    stepType: string = "SWITCH_TAB";
    switchTabMode: SwitchTabMode | string = "FIRST"; // 关闭页面模式
    customIndex?: number = 0; // 自定义关闭页面索引
}